addons/website_event_questions/views/event_views.xml

```diff
@@ -82,7 +82,7 @@
     <field name="inherit_id" ref="event.view_event_registration_form" />
     <field name="arch" type="xml">
         <group name="event" position="inside">
-            <field name="answer_ids" widget="many2many_tags" readonly="1"/>
```

Comment:
Not readonly anymore ? (Not sure to understand the specs)

Reply:
No it is not read only anymore, the admin can now modify/set the attendee answers